src/e2e/resources/jointParty/directionIssuedCallback.json

```diff
@@ -4,7 +4,7 @@
     "callback_response_status": null,
     "callback_response_status_code": null,
     "case_data": {
-      "directionTypeDl": "appealToProceed",
+      "directionTypeDl": "grantExtension",
```

Comment:
Do we need a separate file for each directionTypeDl, maybe not as they all work the same way

Reply:
I'm not sure. If we are to add an e2e test per template we probably should do. Since there are now 3 bits of conditional logic per template there should be again an additional 3. However, since we can't inspect the data in the PDF we send using docmosis and gov notify, I see no point.